docker/build/Dockerfile

```diff
@@ -62,7 +62,7 @@ RUN pip install --allow-unverified pyodbc\
     numpy \
     pyodbc \
     pexpect \
-    virtualenv
+    pandas
```

Comment:
This isn't strictly necessary because you list pandas as a dependency in setup.py.
However, I would keep it to speed things up during tests. Otherwise we'd end up installing pandas every time we `pip install` pyfora.